packages/terra-graphs/package.json

```diff
@@ -18,22 +15,59 @@
     "terra-graphs",
     "carbon",
     "graphs",
-    "UI"
+    "UI",
+    "d3",
+    "d3-module",
+    "graphing"
   ],
   "author": "Cerner Corporation",
   "license": "Apache-2.0",
   "bugs": {
     "url": "https://github.com/cerner/terra-graphs/issues"
   },
   "homepage": "https://github.com/cerner/terra-graphs#readme",
-  "peerDependencies": {
-    "react": "^16.8.5",
-    "react-dom": "^16.8.5"
+  "files": [
+    "dist",
+    "lib",
+    "README.md"
+  ],
+  "prettier": {
+    "arrowParens": "always",
+    "bracketSpacing": true,
+    "semi": true,
+    "tabWidth": 4,
+    "trailingComma": "none",
+    "overrides": [
+      {
+        "files": "package.json",
+        "options": {
+          "tabWidth": 2,
+          "printWidth": 250
+        }
+      }
+    ]
+  },
+  "publishConfig": {
+    "access": "public"
   },
   "dependencies": {
     "classnames": "^2.2.5",
     "prop-types": "^15.5.8",
-    "terra-theme-context": "^1.0.0"
+    "terra-theme-context": "^1.0.0",
+    "d3": "^5.16.0"
+  },
+  "devDependencies": {
+    "@babel/plugin-transform-classes": "^7.10.4",
+    "@commitlint/cli": "^8.3.5",
+    "@commitlint/config-conventional": "^8.3.4",
+    "autoprefixer": "^9.8.4",
+    "conventional-changelog-eslint": "^3.0.8",
+    "jasmine-core": "^3.5.0",
+    "stylelint-config-standard": "^20.0.0"
+  },
```

Comment:
I don't think this package uses these dependencies.

Reply:
removed